test/logflare/log_event_test.exs

```diff
@@ -42,7 +43,8 @@ defmodule Logflare.LogEventTest do
              drop: false,
              # validity gets overwritten
              valid: true,
-             validation_error: "",
+             validation_error: nil,
+             ingest_error: nil,
```

Comment:
would also need to update controller tests to cover rendering of these errors to api clients.

Reply:
Think I got all of them. If not link specific ones and I'll fix.